module_utils/network_lsr/argument_validator.py

```diff
@@ -413,11 +413,23 @@ def _validate_impl(self, value, name):
 
 
 class ArgValidatorMac(ArgValidatorStr):
-    def __init__(self, name, force_len=None, required=False, default_value=None):
+    def __init__(self, name, force_len=[6, 20], required=False, default_value=None):
```

Comment:
Instead of using magic number. Please use constant and make a comment for the constant why 6 and 20 matters.

Reply:
or just don't use this force-len argument. The correct length depends on the layer 2 type, and `ArgValidatorMac` does not know whether it's used in an ethernet or infiniband profile.

The length was already validated at a later pointer, where the knowledge about the layer2 type is present and a good message can be raised (like "and ethernet MAC address must be 6 octets long").

Reply:
I used a constant `MAC_ADDR_OCTETS = [6, 20]` instead. See
https://github.com/linux-system-roles/network/blob/e98e8d1bc64259f711b338771ed12479b7eb091a/module_utils/network_lsr/argument_validator.py#L14